templates/web/fixmystreet.com/report/_report_meta_info.html

```diff
@@ -0,0 +1,5 @@
+[% problem.meta_line(c) | html %]
+[% IF c.cobrand != problem.get_cobrand_logged AND problem.get_cobrand_logged.is_council %]
```

Comment:
Does equality comparison work here? I think you need to compare the moniker or the class name.